scripts/net/README.txt

```diff
@@ -0,0 +1,63 @@
+The shell script run-sample-tests.sh runs a few Zephyr samples against the
+network test applications container provided by the 'net-tools'
+Zephyr project, https://github.com/zephyrproject-rtos/net-tools.
+
+
+Installation
+************
+
+As a prerequisite it is assumed that Docker is installed and that the
+'net-tools' Docker container has been created, see the first bullet point
+at the net-tools Docker README file
+https://github.com/zephyrproject-rtos/net-tools/blob/master/README.docker.
+In essence, the following needs to be done:
+
+ * Install Docker
+ * Check out the net-tools project from github
```

Comment:
net-tools is part of the west manifest, the current sha in west.yml is old and does not have docker/ directory, you need to update west.yml with the latest net-tools and point the user to it, i.e. `../tools/net-tools`. there is no need to checkout a new instance of it.

Reply:
Done.